generated/java/proto-google-common-protos/src/main/proto/google/cloud/audit/audit_log.proto

```diff
@@ -1,128 +0,0 @@
-// Copyright 2016 Google Inc.
```

Comment:
I didn't realize this was in here before - I think it would be a breaking change to remove it

Reply:
Ok, put it back in.